test/NuGet.Core.Tests/NuGet.Commands.Test/NuGet.Commands.Test.csproj

```diff
@@ -10,6 +10,7 @@
 
   <ItemGroup>
     <ProjectReference Include="$(TestUtilitiesDirectory)Test.Utility\Test.Utility.csproj" />
+    <ProjectReference Include="..\..\..\src\NuGet.Clients\NuGet.VisualStudio.Common\NuGet.VisualStudio.Common.csproj" />
```

Comment:
I don't think we should do this. 

NuGet.Commands is a core level test project and shouldn't depend on the VS layer.

Reply:
I believe I have a telemetry dependency on this. Taking a look.

Reply:
I think I can move the test to NuGet.VisualStudio.Common.Test and avoid this project dependency.

Reply:
Sounds good. 

Re-request review when ready.

Reply:
Ended up keeping the test in the same class. I mocked telemetry interface to get rid of telemetry service provided by N.VS.Common